usaspending_api/api_contracts/contracts/v2/disaster/recipient/location/spending.md

```diff
@@ -0,0 +1,163 @@
+                "type": "object"
+            }
+
+    + Attributes
+        + `filter` (required, Filter, fixed-type)
+        + `geo_layer` (required, enum[string], fixed-type)
+            + Members
+                + `state`
+                + `county`
+                + `district`
+        + `pagination` (optional, Pagination, fixed-type)
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, array[Result], fixed-type)
+        + `pagination_metadata` (required, PageMetadata, fixed-type)
+
+
+    + Body
+
+            {
+                "results": [
+                    {
+                        "id": 43,
+                        "code": "ND-01",
+                        "description": "North Dakota Congressional District 1",
+                        "count": 54,
+                        "obligation": 89.01,
+                        "outlay": 76205
+                    },
+                    {
+                        "id": 41,
+                        "code": "ND-02",
+                        "description": "North Dakota Congressional District 2",
+                        "count": 2,
+                        "obligation": 50,
+                        "outlay": 10
+                    }
+                ],
+                "pagination_metadata": {
+                    "page": 1,
+                    "next": 2,
+                    "previous": null,
+                    "hasNext": true,
+                    "hasPrevious": false,
+                    "total": 23,
+                    "limit": 2
+                }
+            }
+
+# Data Structures
+
+## Filter (object)
++ `def_codes` (required, array[DEFC], fixed-type)
++ `fiscal_year` (required, number)
++ `award_type_codes` (optional, array[AwardTypeCodes], fixed-type)
+    Defaults to all Award Type Codes. Applicable only when requested `award` spending.
+
+## Pagination (object)
++ `page` (optional, number)
+    Requested page of results
+    + Default: 1
++ `size` (optional, number)
+    Page Size of results
+    + Default: 10
++ `order` (optional, enum[string])
+    Indicates what direction results should be sorted by. Valid options include asc for ascending order or desc for descending order.
+    + Default: `desc`
+    + Members
+        + `desc`
+        + `asc`
++ `sort` (optional, string)
+    Optional parameter indicating what value results should be sorted by. Valid options are any of the fields in the JSON objects in the response. Defaults to the first field provided.
+
+## Result (object)
++ `id` (required, string)
+    Unique identifier which might be useful for links to resources
++ `code` (required, string)
+    Unique or non-unique value which is the technical name of the result
++ `description` (required, string)
+    Display Text for humans
++ `count` (required, number)
+    Number of Awards
++ `obligation` (required, number, nullable)
++ `outlay` (required, number, nullable)
```

Comment:
May need `face_value_of_loan` here too. Also, worth adding the caveat that this will only have a value when the Loan (and others?) `award_type_code` is provided. In that case, will obligation or outlay have any values?

Reply:
Added new loan endpoint